### DISCUSSION TITLE
en/start/join

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#733] [Announcements]
# en/start/join

Introduction Welcome to our community! To join the Discord server and start using Korepi, follow these steps. Step 1: Create a Discord Account If you don't have a Discord accoun...

https://korepi.com/en/start/join.html

**Comment by malachaistyxx** [comment]
if the you cant get into the discord server becuase it isnt open then the download shouldnt be locked behind it. iv got korepi for kenshin impact wanted to update it but the only place i can find a download is here and i cant even use it becuase im not on a freaking discord server is total crap

**Comment by Malthael11484** [comment]
i cant get into server i dont know why i was there one time but somone from administration banned me for i dont know what i never writed there anything and i just got banned is it the way i can stil get there?

**Reply by luciel2291** [reply]
i just enter into the server for 5 mins, and they banned me. dafuq
